setup.py

```diff
@@ -24,6 +24,7 @@
     install_requires=[
         'Markdown==2.6.2',
         'bleach==1.4.2',
+        'Pygments==2.1.3',
```

Comment:
Having it there doesn't do a lot, until you add the CSS file. I'm ambivalent about whether this stays here and we amend the wording, or whether we remove the requirement, and insert instructions on adding it manually.